### ISSUE TITLE
Userland Sample

### ISSUE THREAD

**Issue by plutocow** [Exaphis/Tsunami#1]
Hello,
I stumbled upon your project while looking for resources to learn kernel development from and I was wondering if you were planning on releasing a sample for usermode?

**Comment by Exaphis** [comment]
Just added, thanks for the suggestion. Out of curiosity, how did you find this project?

**Comment by plutocow** [comment]
Just had a look, looks like a great resource to learn from thank you.
I started getting curious about kernel development so I just searched windows kernel into the top bar and sorted by recently updated. (Yours was last updated 10 hours ago which got me interested!)

I'm looking at the Read method inside of KeInterface.h and I'm wondering, how can it be extended to read a string? I'm trying to wrap my head around all of this but I've read that to read a string, I should first read a char and increase by sizeof(char) and so on until a null terminator is reached. Is there a better way to do this?
